src/Tribe/Aggregator/Record/Queue_Realtime.php

```diff
@@ -196,6 +210,59 @@ public function get_unable_to_continue_processing_data() {
 	 * @return mixed|string|void
 	 */
 	public function get_progress_message_data( $queue, $percentage, $done ) {
+		_deprecated_function( __METHOD__, 'TBD' );
+
+		return wp_json_encode( $this->get_progress_data($queue, $percentage, $done) );
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: cae18937af76e184390c0c441b118e1a</sub>